pyglider/utils.py

```diff
@@ -197,11 +197,14 @@ def get_profiles_new(ds, min_dp = 10.0, inversion=3., filt_time=100,
 
 
 def get_derived_eos_raw(ds):
-    # GPCTD requires a scale factor of 10 for conductivity. Legato does not
-    if 'LEGATO' in ds.conductivity.source:
+    # GPCTD and slocum ctd require a scale factor of 10 for conductivity. Legato does not
+    if 'S m' in ds.conductivity.units:
+        r = 10 * ds.conductivity / seawater.constants.c3515
+    elif 'mS cm' in ds.conductivity.units:
         r = ds.conductivity / seawater.constants.c3515
     else:
-        r = 10 * ds.conductivity / seawater.constants.c3515
+        raise ValueError("Could not parse conductivity units in yaml. Expected 'S m-1' or 'mS cm-1'.\n"
```

Comment:
I'm not sure what conventions we want to have, but the gurus in Matplotlib tell me not to put our own carriage returns in Errors and warnings.  I'm not too fussed if we do this or not, but they claim it improves readability if someone has a narrow terminal or something.

Reply:
Good to follow best practices. CR removed